src/zeep/transports.py

```diff
@@ -183,15 +183,17 @@ def __init__(
 
         self._close_session = False
         self.cache = cache
+        proxy_kwarg_name = "proxy" if httpx.__version__ >= "0.26.0" else "proxies"
```

Comment:
wrong check version, example
```
>>> "0.3.0" >= "0.26.0"
True
```
need to do something like this
```
>>> import packaging.version
>>> packaging.version.parse("0.3.0") >= packaging.version.parse("0.26.0")
False
```

Reply:
Good catch, thank you. I updated the version check.